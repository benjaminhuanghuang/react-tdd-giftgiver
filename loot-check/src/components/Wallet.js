import React, { Component } from 'react'


 class Wallet extends Component {
  render() {
    return (
      <div>
        <h2>Loot Check</h2>
      </div>

    )
  }
}

export default Wallet