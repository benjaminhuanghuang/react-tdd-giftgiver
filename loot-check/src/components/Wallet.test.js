import React from "react"
import { shallow } from 'enzyme'
//
import Wallet from './Wallet'

describe('Wallet', ()=>{
  const app = shallow(<Wallet />)

  it("renders correnctly", () => {
    expect(app).toMatchSnapshot()
  })
  
})
