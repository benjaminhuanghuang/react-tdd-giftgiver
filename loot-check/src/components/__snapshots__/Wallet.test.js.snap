// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correnctly 1`] = `ShallowWrapper {}`;

exports[`Wallet renders correnctly 1`] = `ShallowWrapper {}`;
